cli/cmd/root.go

```diff
@@ -26,7 +26,7 @@ Find more information at:
 func NewCmdRoot() *cobra.Command {
 	logLevel := enum.NewEnum([]string{"trace", "debug", "info", "warn", "error"}, "info")
 	logFormat := enum.NewEnum([]string{"text", "json"}, "text")
-	telemetryLevel := enum.NewEnum([]string{"trace", "debug", "info", "warn", "error"}, "info")
+	telemetryLevel := enum.NewEnum([]string{"none", "errors", "all"}, "all")
```

Comment:
@yevgenypats I think we should reconsider the options here: some people may be okay with sending some anonymous usage data, but not okay with sending errors (since those could in theory contain some identifying information by accident).

Maybe `none`, `errors`, `stats`, `all`?

Reply:
sounds good.